### ISSUE TITLE
No logging when `-c` given no value or if value is incorrect

### ISSUE THREAD

**Issue by mkokho** [Braedon/prometheus-es-exporter#1]
```
$ docker run --rm --name es-exporter -v /tmp/query.cfg:/mnt/configs/exporter.cfg -p 9988:8080 braedon/prometheus-es-exporter:0.1.0 -e mon-es1.movio.co 
[2016-07-21 02:36:11,376] root.INFO MainThread Starting server...
[2016-07-21 02:36:11,376] root.INFO MainThread Server started on port 8080
[2016-07-21 02:36:11,377] root.INFO MainThread Shutting down
```

**Comment by Braedon** [comment]
@mkokho PR here: https://github.com/Braedon/prometheus-es-exporter/pull/2

If you're happy I'll merge and release a new version.

**Comment by Braedon** [comment]
Fix merged. Will be released with 0.1.1
